src/main/java/com/binance/dex/api/client/crosschain/Bep2TokenSymbol.java

```diff
@@ -0,0 +1,23 @@
+package com.binance.dex.api.client.crosschain;
+
+import com.binance.dex.api.client.encoding.ByteUtil;
+import com.binance.dex.api.client.encoding.serializer.Bep2TokenSymbolSerializer;
+import com.binance.dex.api.client.rlp.RlpDecodable;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@JsonSerialize(using = Bep2TokenSymbolSerializer.class)
+public class Bep2TokenSymbol implements RlpDecodable {
```

Comment:
Shall we distinguish BEP2 Token and BEP8 for cross chain tx? If not, then we may use a consolidated name

Reply:
I‘m not sure, but it is named as `Bep2TokenSymbol` on chain. I just think we should keep consistent.

Reply:
Agree with Erhang.  When we write the code, the bep8 is still under development. We change the name into `TokenSymbol ` now.